sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -442,6 +442,32 @@ def test_partial_dependence_sample_weight():
     assert np.corrcoef(pdp, values)[0, 1] > 0.99
 
 
+def test_partial_dependence_sample_weight_hgbt():
```

Comment:
Let's just have a simple `check_hist_gbdt_sw_not_supported` 

No need for the overkill setup if we're not gonna use it in the test